frontend/app/(auth)/login/__tests__/page.test.tsx

```diff
@@ -19,13 +19,17 @@ vi.mock("@/lib/context/SupabaseProvider", () => ({
   useSupabase: () => mockUseSupabase(),
 }));
 
+vi.mock("@/lib/assets/QuivrLogo", () => ({
+  QuivrLogo: () => <div>LOGO</div>,
+}));
+
 vi.mock("@/services/analytics/june/useEventTracking", () => ({
   useEventTracking: () => ({ track: vi.fn() }),
 }));
 
 describe("Login component", () => {
   afterEach(() => {
-    vi.clearAllMocks();
```

Comment:
Why the change ?

Reply:
I was having an error when running tests in parallel while using `vi.clearAllMocks()`. It seems to have a side effect on other tests